### DISCUSSION TITLE
Conky + Thanks

### DISCUSSION THREAD

**Discussion by bee-on-the-green** [kovidgoyal/kitty#6301] [Show and tell]
I have just installed Kitty: it took me less than 1 minute, and everything works great out of the box. 

Now I have a second terminal emulator which is entirely dedicated to my home-made little desktop 'conky' (it's not really a conky, just a window).

A big thank you to kovidgoyal and to all the people who took part in this project!

![image](https://github.com/kovidgoyal/kitty/assets/118056528/275c97ce-bc54-4436-8a8a-f2ca38bc98e5)

**Comment by kovidgoyal** [comment]
You are most welcome!

**Reply by bee-on-the-green** [reply]
:)
